build.gradle.kts

```diff
@@ -48,7 +61,7 @@ kotlin {
         val commonMain by getting {
             dependencies {
                 implementation(kotlin("stdlib-common"))
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-native:$coroutinesVersion")
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:$coroutinesVersion")
```

Comment:
this actually isn't right. the -native artifact has the module file in it, which has the correct references to the js artifact. it's just named -native for legacy/historical reasons

Reply:
yep confirmed this locally. i just added the js target and built, and it resolved dependencies okay — just complained that no actual classes for js (just tested on master without the rest of the changes)